.github/workflows/build-lint-test.yml

```diff
@@ -13,20 +11,29 @@ jobs:
       matrix:
         node-version: [12.x, 14.x, 16.x, 18.x, 20.x]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
```

Comment:
Any reason we are not using v4 for this one?

Reply:
This PR was old huh! And no, updated!